docker.md

```diff
@@ -86,6 +86,13 @@ docker stop [options] CONTAINER
 
 Start/stop a `container`.
 
+#### Example
+
+```
+# Stop all running containers
+$ docker stop $(docker ps -aq)  
+```
```

Comment:
Can you swap the order of the text and examples to match others on this page?

    #### Example
    
    ```
    # Stop all running containers
    $ docker stop $(docker ps -aq)  
    ```
    
    Start/stop a `container`.